initializr-web/src/main/java/io/spring/initializr/web/mapper/InitializrMetadataV22JsonMapper.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2012-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.initializr.web.mapper;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.spring.initializr.metadata.Dependency;
+
+/**
+ * A {@link InitializrMetadataJsonMapper} handling the metadata format for v2.2
+ * <p>
+ * Version 2.2 updates the "versionRange" to "compatibilityRange" for better
+ * clarification.
+ *
+ * @author HaiTao Zhang
+ */
+public class InitializrMetadataV22JsonMapper extends InitializrMetadataV21JsonMapper {
```

Comment:
I don't think we are ready to introduce 2.2 yet. This change should be part of V3

Reply:
Agreed upon grouping multiple changes to V3 instead of having a V2.2 of the `InitializrMetadataJsonMapper`.